### DISCUSSION TITLE
article/shang-yang-bian-fa

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#100] [Announcements]
# article/shang-yang-bian-fa

驭民五术与恩情论，商鞅通过变法，将秦国由一个弱国，真正改造成富裕强大之国，为秦国将来统一六国，建立一统天下的秦朝，起到至关重要的作用。历朝历代对于商鞅进行的富国强兵的变法都有极高的评价。

https://blog.tangly1024.com/article/shang-yang-bian-fa

**Comment by tieyasha** [comment]
不错，值得好好学习

**Reply by tieyasha** [reply]
试了自己评论的，好像是没办法删除
